packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -141,7 +141,8 @@
     "screenIsBeingSharedMessage": "You are sharing your screen",
     "screenShareLoadingMessage": "Loading {participant}'s screen",
     "localVideoLabel": "You",
-    "localVideoCameraSwitcherLabel": "Cycle camera",
-    "localVideoMovementLabel": "Movable Local Video Tile"
+    "localVideoCameraSwitcherLabel": "Switch camera",
+    "localVideoMovementLabel": "Movable Local Video Tile",
+    "localVideoSelectedDescription": "selected"
```

Comment:
nit: Should `s` be capital?

Reply:
The string comes after the camera name. 
Lowercase makes sense in this case. However, we can change it to uppercase for consistency. What do you think?

Would look like:
<Camera name> **selected**.
vs.
<Camera name> **Selected**.

Reply:
I think lowercase will make more sense with the string builder as this string will become something like `{cameraName} selected`

Reply:
Will keep it as lowercase as it makes more sense in this case.